src/Land/Model/soil_heat.jl

```diff
@@ -194,22 +194,17 @@ function compute_gradient_flux!(
         thermal_conductivity(κ_dry, kersten, κ_sat) * ∇transform.soil.heat.T
 end
 
-function flux_second_order!(
-    land::LandModel,
-    soil::SoilModel,
-    heat::SoilHeatModel,
-    flux::Grad,
-    state::Vars,
-    diffusive::Vars,
-    hyperdiffusive::Vars,
-    aux::Vars,
-    t::Real,
-)
+struct DiffHeatFlux <: TendencyDef{Flux{SecondOrder}} end
```

Comment:
I wonder if we might rename these.. These arent fluxes, but divergences of fluxes.  I have also seen people refer to the water driven part as heat advection...will look around the literature and see what names people use for this. I guess the first is `HeatConduction`.

Reply:
I think the phenomenon in general is an acceptable way to look at it. I think `HeatConduction` is probably fine, too. FWIW, I think it's not too difficult to change these names after the fact. Ultimately there will be a function (e.g.,)
```julia
flux(::InternalEnergy, ::DiffHeatFlux, land::LandModel, args)
```
That will return the flux (the divergence of which is taken by the framework and added to all other tendencies).